src/functions.js

```diff
@@ -279,7 +280,7 @@ defineFunction("cr", ["\\\\", "\\cr"], {
     const size = optArgs[0];
     return {
         type: "cr",
-        size: size,
+        size: size && assertNodeType(size, "size"),
```

Comment:
Why `&&`?

Reply:
`size` is nullable since it's an optional argument. `assertNodeType` requires a non-nullable and returns one as well. So, this full expression passes a `?ParseNode<"size">` for the size attribute.

Reply:
Already synced and resolved this. ;)